examples/fedprox_example/server.py

```diff
@@ -54,6 +62,10 @@ def main(config: Dict[str, Any], server_address: str) -> None:
         config["local_epochs"],
         config["batch_size"],
         config["n_server_rounds"],
+        config["adaptive_proximal_weight"],
+        config["proximal_weight"],
+        config["proximal_weight_delta"],
+        config["proximal_weight_patience"],
```

Comment:
I might be missing something, but I removed these from the config, because I didn't think they were used anywhere on the client-side. That is, they are set in the strategy class below in this server. So I didn't think they needed to be sent as part of the client config. I might be wrong though.